src/overlays/actors/ovl_En_Ani/z_en_ani.h

```diff
@@ -28,10 +28,10 @@ typedef struct EnAni {
     /* 0x2FC */ EnAniActionFunc actionFunc;
 } EnAni; // size = 0x300
 
-enum EnAniType {
-  /* else */ ANI_TYPE_STANDING,         // unfinshed and unused
-  /* 1    */ ANI_TYPE_TREE_HANGING = 1,
-};
+typedef enum EnAniType {
+  /* 0 */ ANI_TYPE_STANDING, // unfinshed and unused
```

Comment:
Not really fond of this, sure else isn't standard but its never specified zero anywhere, in the code its always the else clause.

Reply:
Ah, I see what it was going for. Just by including it here though, it's still being forced to take a value by having this enum though. Perhaps the comment can be expanded upon instead explaining this "else" behavior?

Reply:
What do you think of moving ANI_TYPE_STANDING to below TREE_HANGING? I feel like else condition should come after actually used values, and TREE_HANGING already has a hard value.

Reply:
Yeah, fine with me.